### PR TITLE
update to 5.6.1

### PR DESCRIPTION
**Pull request by boldorider4**
* [x] check [license](https://github.com/pymc-devs/pymc/blob/v5.6.1/LICENSE)
* [x] update dependencies according to upstream [requirements](https://github.com/pymc-devs/pymc/blob/v5.6.1/requirements.txt)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,62 +1,71 @@
...
-  sha256: 200a28469d574a370fa9b55cd92f112f8ad09f71674bdae514437039e953689b
+  sha256: a09b11a0ae9a56a6f57b9b89f161d06b1d49b7d80a84311eba4376f2a32b5d0f
 
 build:
   number: 0
+  # skip because arviz is not supported for py38
+  skip: True  # [py<39]
   script:
     - export LDFLAGS="-undefined dynamic_lookup -bundle"  # [osx]
     - export LDFLAGS="$LDFLAGS -shared"  # [linux]
```
Comment (akabanovs) [inline recipe/meta.yaml]: You could use script_env attribute to define env variables, like here:
https://github.com/AnacondaRecipes/flit-feedstock/blob/master/recipe/meta.yaml#L47

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,62 +1,71 @@
-{% set version = "2.3.8" %}
+{% set version = "5.6.0" %}
```
Comment (akabanovs) [inline recipe/meta.yaml]: v 5.6.1 already available

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,62 +1,71 @@
...
 
 package:
   name: pymc
```
Comment (akabanovs) [inline recipe/meta.yaml]: could set variable name (and use here and in the url), although this is not critical.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,62 +1,71 @@
...
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
-  home: http://github.com/pymc-devs/pymc
+  home: https://www.pymc.io/welcome.html
   license: AFL-3.0
   license_family: OTHER
```
Comment (akabanovs) [inline recipe/meta.yaml]: Should not it be "APACHE" here? I think AFL refers to Apache?

File: recipe/meta.yaml (hunk 1 above)
Comment (boldorider4) [reply recipe/meta.yaml]: thanks!

File: recipe/meta.yaml (hunk 2 above)
Comment (boldorider4) [reply recipe/meta.yaml]: yup

File: recipe/meta.yaml (hunk 3 above)
Comment (boldorider4) [reply recipe/meta.yaml]: will do.

File: recipe/meta.yaml (hunk 4 above)
Comment (boldorider4) [reply recipe/meta.yaml]: right, thanks for catching this. I fixed the license as it is Apache-2.0.

File: recipe/meta.yaml [hunk 5]
```diff
@@ -1,62 +1,73 @@
...
-  sha256: 200a28469d574a370fa9b55cd92f112f8ad09f71674bdae514437039e953689b
+  url: https://github.com/pymc-devs/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 3445f91136e44fe02a3e5029b2ececd262bc6623d4622c02cc47da21fa634be2
 
 build:
   number: 0
+  # skip because arviz is not supported for py38
+  skip: True  # [py<39]
+  script_env:
+    - LDFLAGS="-undefined dynamic_lookup -bundle"  # [osx]
```
Comment (lorepirri) [inline recipe/meta.yaml]: `-bundle` seems to create problems (see logs `osx-arm64`)

Comment (boldorider4) [reply recipe/meta.yaml]: Right. Thanks for catching that.

File: abs.yaml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+upload_without_merge: true
+aggregate_check: false
+upload_channels:
+  - boldorider4-test-channel
+channels:
+  - boldorider4-test-channel
```
Comment (cbouss) [inline abs.yaml]:
```suggestion
  - boldorider4-test-channel
build_parameters:
  - "--suppress-variables"
#   - "--skip-existing"
  - "--error-overlinking"
```

File: recipe/meta.yaml [hunk 7]
```diff
@@ -1,62 +1,72 @@
...
-  sha256: 200a28469d574a370fa9b55cd92f112f8ad09f71674bdae514437039e953689b
+  url: https://github.com/pymc-devs/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 3445f91136e44fe02a3e5029b2ececd262bc6623d4622c02cc47da21fa634be2
 
 build:
   number: 0
+  # skip because arviz is not supported for py38
+  skip: True  # [py<39]
+  script_env:
+    - LDFLAGS="-undefined dynamic_lookup"  # [osx]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: As Cheng said the flag should be like this:
`LDFLAGS="${LDFLAGS} -undefined dynamic_lookup"  # [osx]`

**Review by cbouss — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**